src/app/pages/shell/shell.component.ts

```diff
@@ -41,13 +41,15 @@ export class ShellComponent implements OnInit, OnChanges, OnDestroy {
   public resize_terminal = true;
   private shellSubscription: any;
 
-  public shell_tooltip = T('Copy/paste with <b>Ctrl + C/V</b> or\
-   <b>Command + C/V</b>.<br>\
-   Many utilities are built-in, including:<br>\
-   <b>Iperf</b>, <b>Netperf</b>, <b>IOzone</b>, <b>arcsat</b>,\
-   <b>tw_cli</b>, <b>MegaCli</b>,<b>freenas-debug</b>,<b>tmux</b>,\
-   and <b>Dmidecode</b>. See the <b>Guide > Command Line Utilities</b>\
-   chapter for more information.');
+public shell_tooltip = T('<b>Ctrl+C</b> kills a foreground process. <br> \
+                        Many utilities are built-in: <br>\
+                        <b>Iperf</b>, <b>Netperf</b>, <b>IOzone</b>, \
+                        <b>arcsat</b>, <b>tw_cli</b>, <br><b>MegaCli</b>, \
+                        <b>freenas-debug</b>, <b>tmux</b>, <b>Dmidecode</b>. \
+                        <br>\
+                        Refer to the <b>Command Line Utilities</b> chapter in\
+                        the <b>User Guide</b> for usage information and \
```

Comment:
This sentence may need a rework to link to that chapter in the guide, but we're still looking into how to do that kind of link.

Reply:
Good to go - I've got another PR in that has the new syntax.